experiments/storm/src/main/scala/io/gearpump/experiments/storm/producer/StormProducer.scala

```diff
@@ -38,6 +43,9 @@ private[storm] class StormProducer(gearpumpSpout: GearpumpSpout,
 
   override def onStart(startTime: StartTime): Unit = {
     gearpumpSpout.start(startTime)
+    if (gearpumpSpout.ackEnabled) {
```

Comment:
If ack is not enabled, we still need to ack the spout in some way. Otherwise, there will always some pending message in spout memory.